main.c

```diff
@@ -184,27 +173,29 @@ static bool init_emitter(FILE *ts_file)
 	/* Stream start */
 	if (!yaml_stream_start_event_initialize(&event, YAML_UTF8_ENCODING))
 		ERROR_GOTO(emitter_delete,
-				"Failed to initialize YAML stream start event");
+			"Failed to initialize YAML stream start event");
```

Comment:
I would probably keep these (here and below) newline/indent changes in a separate commit in this PR.

Reply:
moved to separate commit